redfish-core/lib/log_services.hpp

```diff
@@ -2176,18 +2351,234 @@ inline void requestRoutesDBusEventLogEntryCollection(App& app)
                 if (filePath != nullptr)
                 {
                     thisEntry["AdditionalDataURI"] =
-                        "/redfish/v1/Systems/system/LogServices/EventLog/Entries/" +
+                        "/redfish/v1/Systems/system/LogServices/EventLog/Entries/" +
+                        std::to_string(*id) + "/attachment";
```

Comment:
```
thisEntry["AdditionalDataURI"] = boost::urls::format(
                        "/redfish/v1/Systems/system/LogServices/EventLog/Entries/{}/attachment",
                        std::to_string(*id) );
````

Reply:
This is upstream this way, so I think I can just leave it?  Theoretically could fix upstream and then pull in, but not sure if it's worth the effort.